test/analysis/SmartDetectorsAnalysisTests/AlertPresentationTests.cs

```diff
@@ -162,51 +153,48 @@ private static void VerifyProperty(List<AlertPropertyLegacy> properties, string
 
         public class TestAlertNoSummary : Alert
         {
-            public TestAlertNoSummary(AlertState alertState = AlertState.Active)
-                : base("Test title", default(ResourceIdentifier), alertState)
+            public TestAlertNoSummary()
+                : base("Test title", default(ResourceIdentifier))
             {
                 this.Value = 22.4;
                 this.MachineName = "strongOne";
             }
 
-            [AlertPredicateProperty]
-            [AlertPresentationProperty(AlertPresentationSection.Property, "CPU increased", InfoBalloon = "CPU increase on machine {MachineName}")]
+            [PredicateProperty]
```

Comment:
If we removed "Alert" prefix from here, should we also do it for AlertPresentationPropertyAttribute?